packages/hw-app-btc/src/newops/psbtv2.ts

```diff
@@ -559,7 +560,7 @@ function uint32LE(n: number): Buffer {
 }
 function uint64LE(n: number): Buffer {
   const b = Buffer.alloc(8);
-  b.writeBigUInt64LE(BigInt(n), 0);
+  b.writeBigUInt64LE(new BigNumber(n), 0);
```

Comment:
is this change correct? i'm not sure writeBigUInt64LE will understand new BigNumber 🤔

Reply:
oh sorry wrong pr